include/deal.II/base/polynomials_rannacher_turek.h

```diff
@@ -63,6 +63,14 @@ class PolynomialsRannacherTurek
   */
   Tensor<2, dim> compute_grad_grad(const unsigned int i,
                                    const Point<dim> &p) const;
+  /** Hessian of gradient of basis function @p i at @p p.
+  */
+  Tensor<3, dim> compute_grad_hessian(const unsigned int i,
+                                      const Point<dim> &p) const;
+  /** Hessian of hessian of basis function @p i at @p p.
+  */
+  Tensor<4, dim> compute_grad_hessian_grad(const unsigned int i,
+                                           const Point<dim> &p) const;
```

Comment:
These are mainly needed in FE_Poly::shape_grad_hessian(), which follows the same design for the existing shape_grad and shape_grad_grad. These latter functions are needed in MappingFEField. I've tried mostly to be as consistent with the implementation of the existing design of lower order derivatives as possible.